api/src/main/java/org/openmrs/module/locationbasedaccess/aop/PatientSearchAdviser.java

```diff
@@ -50,52 +38,7 @@ else if (method.getName().equals("getPatientByUuid")) {
 
     @Override
     public Advice getAdvice() {
-        return new PatientSearchAdvise();
+        return new PatientServiceInterceptor();
```

Comment:
Why this rename?

Reply:
I need to change the private PatientSearchAdvise class to public class due to access by the AOP based teston classes.So I moved to public class in another file, and just changed the name of the classe based on the class content.

Reply:
Our convention has been for getAdvice to return an Advice class. Are you changing that?

Reply:
Yes, but there can be multiple type of advices (BeforeMethod, AfterMethod, and Interceptors).So I thought to attach that with the class Name instead of just mention with Advise.

Reply:
I would refactor using a different ticket and pull request instead of mixing this with work that is simply about adding some unit tests.

Reply:
That's good idea.So let me do it and separate those advice classes.

Reply:
In summary, for any change that is not required by your unit tests, can you do it in a separate ticket and pull request?

Reply:
Yes, I will make it

Reply:
@dkayiwa The changes required for this is added into this PR : #39